src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -620,6 +620,8 @@ internal virtual void PrepareSession()
                     X509Certificate certificate = (X509Certificate)tbCert;
                     WebSession.AddCertificate(certificate);
                 }
+
+                store.Close();
```

Comment:
Why not `using X509Store store = new(StoreName.My, StoreLocation.CurrentUser);` in line 608?

Reply:
Like this ?

Reply:
Yep with the `using X509Store store = ...` you no longer need the explicit `Close()`. It will be done automatically once the store var is out of scope.